telemetry-proxy/telemetry_proxy.go

```diff
@@ -5,18 +5,32 @@ import (
 	"fmt"
 	"io"
 	"log"
+	"os"
+	"time"
 
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials/insecure"
 
 	pb "github.com/cornellrocketryteam/Ground-Software/telemetry-proxy/proto-out" // Replace with your proto package path
+
+	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
+	"github.com/influxdata/influxdb-client-go/v2/api/write"
```

Comment:
Are we using v2 instead of v3? From the README of the v2 package it refers you to the below repo for v3:

[https://github.com/InfluxCommunity/influxdb3-go](https://github.com/InfluxCommunity/influxdb3-go)

Reply:
v3 is currently enterprise only. The open-source version is not stable yet, so we will be using v2. Thanks for checking this though! I hadn't looked at v3 until now